src/v/rpc/simple_protocol.cc

```diff
@@ -103,6 +103,17 @@ simple_protocol::dispatch_method_once(header h, net::server::resources rs) {
       = ssx::spawn_with_gate_then(
           rs.conn_gate(),
           [this, method_id, rs, ctx]() mutable {
+              if (ctx->get_header().version > transport_version::v0) {
```

Comment:
nit: should this / will this eventually be changed to `> transport_version::max_supported`?

Reply:
this was an oversight. thanks, will fix.